### PR TITLE
Adds lizard scream sound effect

### PR DESCRIPTION
**Pull request by super12pl**
## About The Pull Request
Adds 4 new lizard scream sounds. Lizard_scream_1 and lizard_scream_2 by n Beats, lizard_scream_3 by -sihiL and lizard_scream_4 by Lord Saladin.

## Why It's Good For The Game
More sound effects.

## Changelog
:cl:
soundadd: lizards can now scream
/:cl:

### CODE REVIEW

**Comment by LordSaladin** [comment]
haha lizard make funny screams now.

**Comment by TotallyNotChief** [comment]
those are horrifying and #4 sounds like a bear

**Comment by LordSaladin** [comment]
> those are horrifying and #4 sounds like a bear
 anything lower then what i based it on sounds like a bear anyway

**Comment by St0rmC4st3r** [comment]
Change number one to be about twice as fast.
Number two is fine, I guess.
Number three is my favorite! Could be sped up just a little only because it sounds somewhat long.
Number four is too bear-like indeed, but can be fixed by changing pitch about 50% up.

A very-very welcome PR nonetheless. 
-Your local Scalymin.

**Comment by Bestparty** [comment]
Absolutely based, more sounds to hear when I beat the inferior races

File: code/modules/mob/living/carbon/human/emote.dm
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/emote.dm]: Keep the else if

File: code/modules/mob/living/carbon/human/emote.dm
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/emote.dm]: Use else if

**Review by PowerfulBacon — COMMENTED** [review]
Copyright / link to original sound files?

**Comment by super12pl** [comment]
> Copyright / link to original sound files?

https://freesound.org/people/-sihiL/sounds/213842/  with https://creativecommons.org/licenses/by/3.0/ for number 3 and heavily edited number 4
and
https://www.youtube.com/watch?v=RQzI5ldSXkg for number 1 and 2

**Comment by PowerfulBacon** [comment]
requesting crossed review to make sure copyright is cool

**Comment by Crossedfall** [comment]
The YouTube one is a little questionable, since they don't provide any actual license to work off of. They just say that the sounds are royalty free. If you can get a proper OK from the creator or a license that would be associated with their works, I'd feel a lot better. It might honestly be easier to find different sounds that have a compatible license; but, I'll leave that up to you.

**Comment by super12pl** [comment]
email
@Crossedfall

**Review by Crossedfall — APPROVED** [review]
Copyright stuff is good to me

**Comment by LordSaladin** [comment]
merge when

**Review by PowerfulBacon — APPROVED** [review]

**Comment by Moccha-Bee** [comment]
top 4 screams to listen and chill to lizard edition.

**Pull request merged**
